script.js

```diff
@@ -0,0 +1,17 @@
+const {resolve, basename} = require('path');
```

Comment:
I don't think this was meant to be left in

Reply:
Nope, removing